0112-scaling-iot-location-assert-fees.md

```diff
@@ -25,8 +25,8 @@ Hotspot owners pay the onboarding and initial location assertion fees for IOT Da
 ## Stakeholders
 
 - **New IOT Hotspot Owners** who have a new Hotspot in which the maker wallet does not contain appropriate funds to cover the initial assertion will now be able to complete that initial assertion for free.
-- **IOT Hotspot Owners** will be able to assert their location for free once per year.
-- **Data-only IOT Hotspot Owners** will be able to assert their location for free once per day.
+- **IOT Hotspot Owners** will be able to assert their location for free once per year, including their first initial assert.
+- **Data-only IOT Hotspot Owners** will be able to assert their location for free once per day, including their first initial assert.
```

Comment:
```suggestion
- **IOT Hotspot Owners** will be able to assert their location for free once per year, including their first assert.
- **Data-only IOT Hotspot Owners** will be able to assert their location for free once per day, including their initial assert.
```

Reply:
Accept all changes to all comments, good to merge